ocaml/typing/typecore.ml

```diff
@@ -2176,17 +2176,14 @@ let map_fold_cont f xs k =
   List.fold_right (fun x k ys -> f x (fun y -> k (y :: ys)))
     xs (fun ys -> k (List.rev ys)) []
 
-let type_label_a_list loc closed env usage type_lbl_a expected_type lid_a_list =
+let disambiguate_sort_lid_a_list loc closed env usage expected_type lid_a_list =
   let lbl_a_list =
     disambiguate_lid_a_list loc closed env usage expected_type lid_a_list
```

Comment:
This is the only use of `disambiguate_lid_a_list`. Maybe just push the sorting code into `disambiguate_lid_a_list` and have get rid of `disambiguate_sort_lid_a_list`.